Source/Moq.csproj

```diff
@@ -28,9 +28,10 @@
 
 	<ItemGroup>
 		<PackageReference Include="Castle.Core" Version="4.2.1" />
-		<PackageReference Include="GitInfo" Version="1.1.14" />
+    <PackageReference Include="GitInfo" Version="1.1.14" />
 		<PackageReference Include="IFluentInterface" Version="2.0.0" />
 		<PackageReference Include="SourceLink.Create.CommandLine" Version="2.2.1" PrivateAssets="All" />
+		<PackageReference Include="System.Threading.Tasks.Extensions" Version="4.4.0" />
```

Comment:
Shouldn't the same package reference be added for the `net46` target in both test projects? I suppose the test projects right now compile only because they reference the Moq project, which in turns references `System.Threading.Tasks.Extensions.dll`. I think it would be more correct if the test projects referenced their required NuGet package directly.

(However, the same reference isn't needed for `netcoreapp2.0`, as the SDK already includes it.)

Reply:
The reason why it "just works" without the explicit reference is because of the new style project file format automatically resolves transitive dependencies. In VS, expand the dependencies node -> .NetFramework 4.6 -> Projects -> Moq and you'll see System.Threading.Tasks.Extensions is indeed referenced by the unit test projects.

I can go ahead and add those references to the csproj and vbproj unit test files since you asked though.

Reply:
Hmmmm, OK, if it's officially documented to work that way perhaps it just takes some getting used to on my part. :-) Thanks for teaching me something new. Both is fine with me then, it's your call!

Reply:
No problem! I think I'll prefer the transitive dependency resolution then. Seems to be the more modern approach. See also [Dotnet Project System](/dotnet/project-system/issues/199); it's a feature pulled from the project.json/xproj days of the early .Net Core SDK and CLI tools.